recipe/build.sh

```diff
@@ -12,4 +12,12 @@ sed -i.bak "s/@rust_default_cc@/$rust_default_cc/g" activate.sh
 sed -i.bak "s/@cross_target_platform@/$cross_target_platform/g" activate.sh
 sed -i.bak "s/@CONDA_RUST_HOST_LOWER@/$(echo $rust_arch_env_build | tr '[:upper:]' '[:lower:]')/g" activate.sh
 sed -i.bak "s/@CONDA_RUST_TARGET_LOWER@/$(echo $rust_arch_env | tr '[:upper:]' '[:lower:]')/g" activate.sh
+
+# cross-compiling
+if [[ "${target_platform}" != "${cross_target_platform}" ]]; then
+  # when cross compiling we must set `CARGO_TARGET_$rust_arch_env_build_RUSTFLAGS`
+  echo 'export CARGO_TARGET_${rust_arch_env_build}_RUSTFLAGS="-C link-arg=-Wl,-rpath,\${BUILD_PREFIX:-\${CONDA_PREFIX}}/lib"' >> activate.sh
```

Comment:
We can actually move this to activate.sh by checking `@CONDA_RUST_HOST_LOWER@` is not equal to `@CONDA_RUST_TARGET_LOWER@`

Reply:
But a "fixed" if-condition will be hard-coded to `activate.sh` no? Don't you prefer having that logic in the build script?

I don't really mind, but just wondering what is the idea. LMK what you prefer.

Reply:
> But a "fixed" if-condition will be hard-coded to activate.sh no?

Yes, but that's preferred since all the logic will be in one place and makes it easier to read.

Reply:
ok got it.

LGTM on my side.